src/soap/serializers/query.js

```diff
@@ -10,7 +10,7 @@ export function queryInfoEnd() {
 
 export function queryInfoType({ method, parameters }) {
   return (
-    `<creationTime>${new Date().toISOString()}</creationTime>` +
+    `<creationTime>${new Date(1661964219333).toISOString()}</creationTime>` +
```

Comment:
I think there was some confusion on this. We would not want to modify the source in this manner to accommodate a test case. Maybe change the signature to something like:

```
export function queryInfoType({ date = new Date(), method, parameters } = {}) {
  return (
    `<creationTime>${date.toISOString()}</creationTime>` +
```

Then pass in a date in the test case.

Reply:
okay Scott, I will change and push